plugins/languages/sniprun.nix

```diff
@@ -9,59 +9,74 @@ with lib;
 let
   cfg = config.plugins.sniprun;
 
-  mkList = helpers.defaultNullOpts.mkNullable (types.listOf types.str);
+  inherit (helpers.defaultNullOpts) mkListOf' mkListOf mkAttrsOf;
```

Comment:
Not a big fan of this. Let's be consistent with the rest of the code base.

Reply:
Moved to #1716